stdlib/REPL/src/REPLCompletions.jl

```diff
@@ -515,20 +517,20 @@ function get_type(T, found::Bool, default_any::Bool)
 end
 
 # Method completion on function call expression that look like :(max(1))
-MAX_METHOD_COMPLETIONS = 40
+const MAX_METHOD_COMPLETIONS = Ref(40)
```

Comment:
If we configure this value, it feels "better" if we can do `REPL.REPLCompletions.MAX_METHOD_COMPLETIONS[] = 100` instead of `@eval REPL.REPLCompletions MAX_METHOD_COMPLETIONS = 100`.

Reply:
It is not really configurable. It is a function argument though, if desired.

Reply:
Then we basically need to copy and paste the definitions of `complete_methods` etc. If it's not really supposed to be configured, we should probably make it `const`.

Reply:
Just because we make it less convenient to change, does not mean we have to make it `const`

Reply:
If people eventually do need to change it regularly, this will let them try out the idea, then add a contextual parameters object to this code to encapsulate any configurations of interest

Reply:
gotcha, let's just remove this change.